protocol/protobuf/pairing.proto

```diff
@@ -298,3 +298,34 @@ message SyncRawMessage {
   bytes subAccountsJsonBytes = 2;
   bytes settingsJsonBytes = 3;
 }
+
+message SyncKeycard {
+  string uid = 1;
+  string name = 2;
+  bool locked = 3;
+  string key_uid = 4;
+  repeated bytes addresses = 5;
+  uint64 last_update_clock = 6;
+}
+
+message SyncKeycardAction {
+  Action action = 1;
+  string oldKeycardUid = 2;
+  SyncKeycard keycard = 3;
+
+  enum Action {
+    KEYCARD_ADDED = 0;
+    ACCOUNTS_ADDED = 1;
+    KEYCARD_DELETED = 2;
+    ACCOUNTS_REMOVED = 3;
+    LOCKED = 4;
+    UNLOCKED = 5;
+    UID_UPDATED = 6;
+    NAME_CHANGED = 7;
+  }
+}
+
+message SyncAllKeycards {
+  repeated SyncKeycard keycards = 1;
+  uint64 syncing_clock = 2;
```

Comment:
This is me being nitpicky, so comment could be ignored, but could the `*_clock` attributes be named `clock`, similar to how it's done in other .proto files?